### PR TITLE
Fixes the Malf borg machine's conveyors not actually moving things (or people)

### PR DESCRIPTION
**Pull request by zxaber**
## About The Pull Request
Adds a line to the conveyors used by the borg machine that enables their processing during Initialize(). Presumably, this was overlooked during the machine processing changes a few months ago. Without this, the conveyors visually turn but don't move anything.

## Why It's Good For The Game
Bug bad. 

## Changelog
:cl:
fix: Borg machine conveyance has been repaired, to the delight of any "alternative"-functioning AIs that wish to onboard new silicon employees.
/:cl:

### CODE REVIEW

**Comment by wabash1** [comment]
Good, it was nearly impossible to convert people before

File: code/modules/recycling/conveyor2.dm [hunk 1]
```diff
@@ -41,6 +41,7 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	. = ..()
 	operating = TRUE
 	update_move_direction()
+	begin_processing()
```
Comment (ShizCalev) [inline code/modules/recycling/conveyor2.dm:44]: needs update_icon() too

Comment (ShizCalev) [inline code/modules/recycling/conveyor2.dm:44]: change the subtype's processing_flags to START_PROCESSING_ON_INIT instead

also, update icon

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/recycling/conveyor2.dm [hunk 2]
```diff
@@ -41,6 +41,7 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	. = ..()
 	operating = TRUE
 	update_move_direction()
```
Comment (ShizCalev) [inline code/modules/recycling/conveyor2.dm:43]: this is already done by the parent.

File: code/modules/recycling/conveyor2.dm [hunk 3]
```diff
@@ -41,6 +41,7 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	. = ..()
 	operating = TRUE
```
Comment (ShizCalev) [inline code/modules/recycling/conveyor2.dm:42]: set this before . = ..()

File: code/modules/recycling/conveyor2.dm (hunk 1 above)
Comment (zxaber) [reply code/modules/recycling/conveyor2.dm]: All set

**Pull request merged**